app/models/film.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+class Film < ApplicationRecord
+  MOVIE = 'Movie'.freeze
+  TV_SERIES = 'TV Show'.freeze
```

Comment:
Really nice practice to store these important fields as constants and freeze the string. I wonder if I wouldn't have added a `_TYPE` suffix to make it clear which column it refers to.

Reply:
Will update this 👍